certificate-policy.md

```diff
@@ -656,50 +656,60 @@ No stipulation.
 See Section 9.6.3, provisions 2. and 4.
 
 ### 4.5.2 Relying party public key and certificate usage
-No stipulation.
+Certificates may specify restrictions on use through critical certificate extensions, including the basic constraints and key usage extensions. 
+
+All CAs operating under this policy provide revocation information in accordance with Section 4.9.7 and Section 4.9.9. 
+
+It is recommended that relying parties process and comply with this information whenever using certificates in a transaction.
 
 ## 4.6 Certificate renewal
 
 ### 4.6.1 Circumstance for certificate renewal
-No stipulation.
+CA and Subscriber certificates issued under the policy SHALL NOT be renewed.
+On-Line Certificate Status Protocol (OCSP) Delegated responder certificates MAY be renewed.
 
 ### 4.6.2 Who may request renewal
-No stipulation.
+TODO
 
 ### 4.6.3 Processing certificate renewal requests
-No stipulation.
+TODO
 
 ### 4.6.4 Notification of new certificate issuance to subscriber
-No stipulation.
+See Section 4.3.2.
 
 ### 4.6.5 Conduct constituting acceptance of a renewal certificate
-No stipulation.
+See Section 4.4.1.
 
 ### 4.6.6 Publication of the renewal certificate by the CA
-No stipulation.
+See Section 4.4.2.
 
 ### 4.6.7 Notification of certificate issuance by the CA to other entities
-No stipulation.
+See Section 4.4.3.
 
 ## 4.7 Certificate re-key
+Once a certificate has been rekeyed, the superseded certificate MAY or MAY NOT be revoked, but SHALL NOT be further re-keyed or modified.
+Subscribers SHALL identify themselves for the purpose of re-keying as required in Section 3.3.
 
 ### 4.7.1 Circumstance for certificate re-key
-No stipulation.
+TODO 
 
 ### 4.7.2 Who may request certification of a new public key
-No stipulation.
+Requests for certification of a new public key SHALL be considered as follows:  
+•	Subscribers with a currently valid certificate MAY request certification of a new public key  
+•	Authorized organization representative, CAs and RAs MAY request certification of a new public key on behalf of a subscriber.  
```

Comment:
While I apologize if I missed the opportunity to discuss this on our call today, this is an area I still want to discuss, and I laid out reasons here: https://github.com/uspki/policies/issues/48#issuecomment-273609022

@lachellel et al -- Is this still an open area for discussion, or should we discuss it on this PR before merge?